helix-core/src/test/java/org/apache/helix/integration/spectator/TestRoutingTableProviderFromTargetEV.java

```diff
@@ -113,13 +113,17 @@ public void afterClass() throws Exception {
     }
     deleteCluster(CLUSTER_NAME);
   }
+  @Test
+  public void testHead() {
+    Assert.assertTrue(true);
+  }
```

Comment:
Why? I guess you are playing some trick. Could you please explain?

Reply:
Assign an order, not use testHead